src/components/browse/singleDatasetView/singleDatasetViewAccordion/singleDatasetViewAccordion.tsx

```diff
@@ -19,9 +19,9 @@ const SingleDatasetViewAccordion = (props: SingleDatasetViewAccordionProps) => {
           <Table bordered hover className="fs-8" size="sm">
             <thead>
               <tr>
-                <th className="fs-7 w-25">Experiment</th>
-                <th className="w-25">Alias</th>
-                <th>Number of Samples</th>
+                <th className="fs-7 w-25">Experiment ID</th>
```

Comment:
"Down" icon should be white color

Reply:
That would require a lot of coding and would probably need to be a separate task.

Reply:
Please check the design for the Experiment, Sample, and File summary. There should be one rounded box for each.

Reply:
Done